Initial work on Phace Page

samtechie opened the pull request:
These are my proposed next steps

- I was just beginning to work on the Cycling of Phaces . Here I just need to update the address by generating a random address for now.
- Then I would go to Saving a phace here am thinking of having a Type that holds a List of Phaces the user saves.
-  Then I would try to improve on the Layout and replace placeholder text with icons etc.

Most of the time was spent on familiarizing myself a bit with the codebase and I haven't yet figured out how to enable debug mode.

samtechie committed:
Add Initial Phace Page

samtechie committed:
Add Cycle Phace Type

Add CyclePhace Type message and some minor layout changes

samtechie committed:
Merge branch 'team-toast:master' into master

ronanyeah commented:
@samtechie Debug flag: https://github.com/team-toast/SmokeSignal/blob/master/webpack.config.js#L33

ronanyeah commented on src/elm/View/Phace.elm:
```diff
@@ -0,0 +1,79 @@
+        [ width fill
+        , spacingXY 0 20
+        ]
+        [ column
```
A column is not required here because there is only one child.

ronanyeah commented on src/elm/View/Phace.elm:
```diff
@@ -0,0 +1,79 @@
+                        [ el [ centerX, Element.Font.color white ] <| text "SavePhace" ]
+                    ]
+                ]
+            , row [ padding 20 ]
```
This row is also redundant.

ronanyeah commented on src/elm/View/Phace.elm:
```diff
@@ -0,0 +1,79 @@
+                , column [ paddingXY 20 10, spacingXY 10 10 ]
+                    [ row
+                        []
+                        [ el [ centerX, Element.Font.color white, Element.Events.onClick CyclePhace ] <| text "Cycle Phace" ]
```
[Input.button](https://package.elm-lang.org/packages/mdgriffith/elm-ui/latest/Element-Input#button) should be used instead of `Events.onClick`.

ronanyeah commented on src/elm/View/Phace.elm:
```diff
@@ -0,0 +1,79 @@
+                [ width fill ]
+                [ column []
+                    [ phaceEl ]
+                , column [ paddingXY 20 10, spacingXY 10 10 ]
```
`spacingXY 10 10` is equivalent to `spacing 10`.

samtechie committed:
Merge branch 'master' of https://github.com/team-toast/SmokeSignal into team-toast-master

 Conflicts:
	package-lock.json
	src/elm/Routing.elm
	src/elm/Types.elm

samtechie committed:
Merge branch 'team-toast-master'

Fixing merge conflicts from master

samtechie committed:
Generate simple ethereum address

Testing Cycling through phaces to generate an Eth addres. Building on this to continue work on the in Dapp wallet

samtechie committed:
Generate address on Xdai chain

samtechie committed:
Initial work on InDapp transactions

samtechie committed:
In Dapp Wallet Test implementation
